pattern-1/bosh-release/README.md

```diff
@@ -1,87 +1,241 @@
+
+    - Following table shows the external product database configurations, which have been set as properties under WSO2 Identity Server job specifications
+    (**e.g.** see `properties` section under `<pivotal-cf-is>/pattern-1/bosh-release/jobs/wso2is_<job_number>/spec`).
+    
+   <br>
+   
+   Property | Description | Default
+   -------- | ----------- | -------
+   wso2is.mysql.host | Hostname/IP of the MySQL server in which WSO2 Identity Server product database resides. | 192.168.50.1
+   wso2is.mysql.product_db | Name of the WSO2 Identity Server product database. | wso2is_db
+   wso2is.product_db.username | Username of the WSO2 Identity Server product database user. | root
+   wso2is.product_db.password | Password of the WSO2 Identity Server product database user. | root
+      
+   If you customize any of the above configurations in following steps, change the default property values to customized values in each job specification.
+    
+   - Create the product database. For this purpose, execute the `<pivotal-cf-is>/pattern-1/bosh-release/dbscripts/mysql.sql` script.
+        
+    ```
+    DROP DATABASE IF EXISTS <wso2is.mysql.product_db>; CREATE DATABASE <wso2is.mysql.product_db>;
+    USE <wso2is.mysql.product_db>; SOURCE /dbscripts/mysql.sql;
+    ```
+   
+   This will create the tables to hold user management data, identity related data and workflow feature data
+   (see [Setting Up Separate Databases for Clustering](https://docs.wso2.com/display/IS541/Setting+Up+Separate+Databases+for+Clustering)).
+
+2. Move to the root directory of deployment pattern 1 BOSH release.
+
+    ```
+    cd <pivotal-cf-is>/pattern-1/bosh-release
     ```
-    bosh -e vbox vms
-    ...
     
-    Deployment 'wso2is'
+3. Upload the deployment pattern 1 BOSH release.
+
+    ```
+    bosh -e <environment-alias> upload-release
+    ```
+
+4. Upload the desired stemcell directly to BOSH. [bosh.io](http://bosh.io/stemcells) provides a resource to find and download stemcells.
+
+    ```
+    bosh -e <environment-alias> upload-stemcell <URL/local_system_path_to_stemcell>
+    ```
+
+    **e.g.** When uploading the stemcell to BOSH Lite environment created in previous section (see example steps under section [Create the BOSH Release](#create-the-bosh-release)),
+   
+    ```
+    bosh -e vbox upload-stemcell https://bosh.io/d/stemcells/bosh-warden-boshlite-ubuntu-trusty-go_agent
+    ```
     
-    Instance                                       Process State  AZ  IPs          VM CID                                VM Type  
-    wso2is_1/b277265a-6f11-4b59-a82d-87e14b01f898  running        -   10.244.15.2  c5a75be1-8acd-4d09-7a99-4cec2795ed15  wso2is-resource-pool  
-    wso2is_2/1030d639-4b02-48e5-83b0-0ca6ed79fecb  running        -   10.244.15.3  f10c0fc0-7573-4fde-673f-9b6d25ef124c  wso2is-resource-pool  
+5. Upload the deployment manifest.
+
+    ```
+    bosh -e <environment-alias> -d wso2is-pattern-1 deploy manifests/wso2is-manifest.yml
```

Comment:
Currently, we keep IaaS specific details in manifest which means above manifest is bosh lite specific. In that case, just changing the environment alias would not work for here.

Reply:
The necessary refinements have been done in https://github.com/wso2/pivotal-cf-is/pull/44.